serving/docker/tensorrt-llm.Dockerfile

```diff
@@ -77,12 +78,6 @@ RUN pip install torch==${TORCH_VERSION} transformers==${transformers_version} ac
     transformers_stream_generator einops tiktoken jinja2 graphviz blobfile colored h5py strenum pulp flax easydict && \
     pip3 cache purge
 
-# Install TensorRT and TRT-LLM Deps
```

Comment:
The trt installation has been removed, can we document why? Is this just for dry run? If it ships with TRTLLM wheel etc.., then we don't need `tensorrtlibs_version` above anymore because it is not used. How about ammo?

Reply:
That is my bad; fixed